main/src/org/destinationsol/menu/InputMapScreen.java

```diff
@@ -279,33 +282,30 @@ public boolean reactsToClickOutside() {
     }
 
     @Override
-    public boolean isCursorOnBg(SolInputManager.Ptr ptr) {
+    public boolean isCursorOnBg(SolInputManager.Ptr pointers) {
```

Comment:
Plural? `pointers` instead of `pointer` ?

Odd class in general but I suspect it'll get a lot of refactoring later :-)